### DISCUSSION TITLE
SN Discord?

### DISCUSSION THREAD

**Discussion by tsmith123** [stackernews/stacker.news#1465] [General]
Is this the official dev chat for SN or is there a Discord anywhere?

**Comment by riccardobl** [comment]
Hi, there is a self-hosted matrix instance, you need to ask to join [here](https://github.com/stackernews/stacker.news/discussions/1059) .

https://github.com/stackernews/stacker.news/blob/4532e00085f57ae668dd6bfdb73b10a8c98d4ccd/README.md?plain=1#L321-L322

**Reply by tsmith123** [reply]
Thanks. I was invited a while ago but I don't know what the link is

**Reply by riccardobl** [reply]
https://chat.sndev.team/ :+1:

**Reply by tsmith123** [reply]
Thanks for that 👍
